src/test/java/org/commonwl/view/researchobject/ROBundleServiceTest.java

```diff
@@ -136,7 +145,7 @@ public void generateAndSaveROBundle() throws Exception {
         // Check cwl aggregation information
         PathMetadata cwlAggregate = manifest.getAggregation(
                 bundleRoot.resolve("lobSTR-workflow.cwl"));
-        assertEquals("https://raw.githubusercontent.com/common-workflow-language/workflows/933bf2a1a1cce32d88f88f136275535da9df0954/lobstr-draft3/lobSTR-workflow.cwl",
+        assertEquals("https://w3id.org/cwl/v/git/null/lobstr-draft3/lobSTR-workflow.cwl?format=raw",
```

Comment:
Could this accidentally be `null` for other reasons? Rather mock a fake value like `deadbeef1337`